src/Examples/Flowsheets/power_generation/rsofc/CO2_H2O_Ideal_VLE.py

```diff
@@ -0,0 +1,398 @@
+                    * pyunits.K ** -1
+                    * pyunits.kiloK ** -1,
+                ),
+                "C": (
+                    -1.464191e-07,
+                    pyunits.J
+                    * pyunits.mol ** -1
+                    * pyunits.K ** -1
+                    * pyunits.kiloK ** -2,
+                ),
+                "D": (
+                    1.092131e-08,
+                    pyunits.J
+                    * pyunits.mol ** -1
+                    * pyunits.K ** -1
+                    * pyunits.kiloK ** -3,
+                ),
+                "E": (
+                    -3.661371e-08,
+                    pyunits.J
+                    * pyunits.mol ** -1
+                    * pyunits.K ** -1
+                    * pyunits.kiloK ** 2,
+                ),
+                "F": (-6.19735, pyunits.kJ / pyunits.mol),
+                "G": (179.999, pyunits.J / pyunits.mol / pyunits.K),
+                "H": (0, pyunits.kJ / pyunits.mol),
+            },
+            "enth_mol_form_vap_comp_ref": (0, pyunits.J / pyunits.mol),  # [1]
+        },
+    },
+}
+
+
+# returns a configuration dictionary for the list of specified components
+def get_prop(components=None, phases="Vap", eos=EosType.IDEAL, scaled=False):
+    if components is None:
+        components = list(_component_params.keys())
+    configuration = {
+        "components": {},  # fill in later based on selected components
+        "parameter_data": {},
+        "phases": {},
+        # Set base units of measurement
+        "base_units": {
+            "time": pyunits.s,
+            "length": pyunits.m,
+            "mass": pyunits.kg,
+            "amount": pyunits.mol,
+            "temperature": pyunits.K,
+        },
+        # Specifying state definition
+        "state_definition": FTPx,
+        "state_bounds": {
+            "flow_mol": (0, 8000, 1e6, pyunits.mol / pyunits.s),
+            "temperature": (273.15, 500, 3000, pyunits.K),
+            "pressure": (5e4, 1.3e5, 1e8, pyunits.Pa),
+        },
+        "pressure_ref": (101325, pyunits.Pa),
+        "temperature_ref": (298.15, pyunits.K),
+    }
+
+    c = configuration["components"]
+    for comp in components:
+        c[comp] = copy.deepcopy(_component_params[comp])
+    phases = phases
+    for k in phases:
+        if eos == EosType.PR:
+            configuration["phases"][k] = copy.deepcopy(_phase_dicts_pr[k])
+        elif eos == EosType.IDEAL:
+            configuration["phases"][k] = copy.deepcopy(_phase_dicts_ideal[k])
+        else:
+            raise ValueError("Invalid EoS.")
+    if len(phases) > 1:
+        p = tuple(phases)
+        configuration["phases_in_equilibrium"] = [p]
+        configuration["phase_equilibrium_state"] = {p: SmoothVLE}
+
+    # Fill the binary parameters with zeros.
+    d = configuration["parameter_data"]
+    d["PR_kappa"] = {(a, b): 0 for a in c for b in c}
+
+    # # Change to scaled units if specified
```

Comment:
Commented code should probably be removed.

Reply:
Rather than remove the code, would it be preferable to uncomment it and leave the argument `scaled=False`? It is not currently utilized by the rest of the code, but may be a useful scaling in the future seeing as we did something similar to the natural gas property packages.

Reply:
This is a mute point now, as @AlexNoring recommends removing this file.